spark/dl/src/main/scala/com/intel/analytics/bigdl/models/resnet/ResNet.scala

```diff
@@ -42,7 +42,7 @@ object Convolution {
      padH: Int = 0,
      nGroup: Int = 1,
      propagateBack: Boolean = true,
-     initMethod: InitializationMethod = Default,
+     initMethod: Initializer = RandomUniform,
```

Comment:
How about ``` initMethod``` accept a method like this``` (shape: Array[Int], dimOrder: String) => Tensor[T] ``` or  ```(shape: Array[Int]) => Tensor[T]``` as dim_ordering should be fix within BigDL.
and then ```reset() ``` can set weight base on that value.
For bias, it always be filled with zero, so here its more about how to set weight only. cc @yiheng @jason-dai